src/ui/back-button.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import IconArrow from "./icons/icon-arrow.svelte";
+  import Localized from "./localization/localized.svelte";
+  import { LocalizationKeys } from "./localization/supportedLanguages";
+
+  export let disabled = false;
+</script>
+
+<button on:click class="rcb-back-button" {disabled} data-testid="close-button">
+  <IconArrow className="back" />
+  <div>
+    <Localized key={LocalizationKeys.NavbarBackButton} />
```

Comment:
where is font-size defined here?

Reply:
It's inherited by the button, I'll set it explicitly.

Reply:
Done